test_runner/fixtures/neon_fixtures.py

```diff
@@ -3158,6 +3158,67 @@ def __exit__(
         pass
 
 
+class RemoteNeonApiEndpoint:
```

Comment:
I would put this class definition in the noen_api.py file.

Reply:
and maybe just call it RemoteEndpoint

Reply:
We already have a `RemotePostgres`, which may or may not be Neon, so I wanted the name to specify that it was based on `NeonApi`. I'll shorten it to `NeonApiEndpoint`